### DISCUSSION TITLE
Turbo remote caching not working " cache bypass, force executing"

### DISCUSSION THREAD

**Discussion by praveenn77** [vercel/turborepo#9430] [Help] [answered]
### Summary

I tried to add remote caching to my azure pipeline setup, but it is not working.
I can see that the remote cache is enabled but, " cache bypass, force executing"  is happening
<img width="620" alt="image" src="https://github.com/user-attachments/assets/65a8f71a-e348-4ca1-9faf-e1ec16b8d386">

**Comment by anthonyshew** [comment, marked as answer]
This indicates that execution is being forced one way or another. Ways that force can be applied are:
- Via the `TURBO_FORCE` environment variable: https://turbo.build/repo/docs/reference/system-environment-variables
- Via the `--force` flag: https://turbo.build/repo/docs/reference/run#--force

**Comment by praveenn77** [comment]
I found the issue, I used --filter and ran only for few packages
